### PR TITLE
fix count error in cpu_stats

### PR DESCRIPTION
**Pull request by ZongqingQi**
Signed-off-by: zonqi <zonqi@redhat.com>

### CODE REVIEW

**Comment by ZongqingQi** [comment]
Last time I checked the number of CPU, I didn't take into account the possible problems caused by the malformed option in the error test. Now fixed.
[root@localhost tp-libvirt]# avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_stats.negative_test.invalid_start_option5
JOB ID     : be94e2e1b52c81809a0dac56d654105c1bb5c95d
JOB LOG    : /root/avocado/job-results/job-2018-04-20T10.52-be94e2e/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.cpu_stats.negative_test.invalid_start_option5: PASS (82.32 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 85.39 s

**Comment by ZongqingQi** [comment]
The new test failure cases can all pass,based on this modify.
[root@localhost domain]# avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_stats.negative_test.invalid_start_alpha
JOB ID     : a855d1ff58c75df4dcfcb669483cc8d53e572b7f
JOB LOG    : /root/avocado/job-results/job-2018-04-23T10.31-a855d1f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.cpu_stats.negative_test.invalid_start_alpha: PASS (136.53 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 140.25 s
[root@localhost domain]# avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_stats.negative_test.invalid_start_whitespace
JOB ID     : 260d2e44d0105bf27adde579fd384ef5fb24e606
JOB LOG    : /root/avocado/job-results/job-2018-04-23T10.34-260d2e4/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.cpu_stats.negative_test.invalid_start_whitespace: PASS (141.77 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 145.19 s
[root@localhost domain]# avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_stats.negative_test.invalid_start_none
JOB ID     : c1be2ab6f4c089b0670d194749a9eef6bcaac1cb
JOB LOG    : /root/avocado/job-results/job-2018-04-23T10.37-c1be2ab/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.cpu_stats.negative_test.invalid_start_none: PASS (196.67 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 200.15 s
[root@localhost domain]# avocado run --vt-type libvirt --vt-machine-type i440fx timer_management.specific_timer.kvmclock.present_yes.offset_timezone.no_stress.test_specific_timer
JOB ID     : 2bace0f4679d71735ab14c7ef78c740d3f789404
JOB LOG    : /root/avocado/job-results/job-2018-04-23T10.41-2bace0f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.timer_management.specific_timer.kvmclock.present_yes.offset_timezone.no_stress.test_specific_timer: PASS (147.67 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 151.40 s
[root@localhost domain]# avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_stats.negative_test.invalid_start_option5
JOB ID     : 4d4e6452a433655e651239a9f9310add72fbeb03
JOB LOG    : /root/avocado/job-results/job-2018-04-23T10.48-4d4e645/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.cpu_stats.negative_test.invalid_start_option5: PASS (69.91 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 73.04 s

File: libvirt/tests/src/virsh_cmd/domain/virsh_cpu_stats.py [hunk 1]
```diff
@@ -59,9 +59,9 @@ def run(test, params, env):
                 option_dict[match.split(' ')[0]] = match.split(' ')[1]
 
     # check if cpu is enough,if not cancel the test
-    cpu_start = int(option_dict.get("start", "-1"))
-    if not (cpu_start == -1) and (status_error == "no"):
-        if (cpu_start > cpus):
+    if (status_error == "no"):
+        cpu_start = int(option_dict.get("start", "-1"))
+        if not (cpu_start == -1) and (cpu_start >= cpus):
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_cpu_stats.py]: I think cpu_start >= cpus already cover the situation "not (cpu_start == -1)"?

**Comment by kylazhang** [comment]
@ZongqingQi please merge your 2 commits into 1 by #git rebase master --interactive --autosquash

**Comment by ZongqingQi** [comment]
@kylazhang Sorry I am at school today. Do we have some way to merge 2 commit on github?

**Comment by ZongqingQi** [comment]
@kylazhang  I think it is ok now. I fetch the code to my local working branch,and push again.

**Pull request merged**